tests/registry_disk_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	"flag"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/virt-controller/services"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("RegistryDisk", func() {
+
+	flag.Parse()
+
+	coreClient, err := kubecli.Get()
+	tests.PanicOnError(err)
+
+	restClient, err := kubecli.GetRESTClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		tests.BeforeTestCleanup()
+	})
+
+	LaunchVM := func(vm *v1.VM) runtime.Object {
```

Comment:
I didn't find one. Most of the other tests are doing a synchronized "launch this vm and wait for it". This function is different. We want to launch multiple VMs in parallel.  That logic doesn't exist anywhere else in the test suite yet.

If we end up reusing this type of logic outside of registry disks, we can make it generic.